naturallanguageunderstandingv1/natural_language_understanding_v1.go

```diff
@@ -188,8 +197,8 @@ func (naturalLanguageUnderstanding *NaturalLanguageUnderstandingV1) GetDeleteMod
 }
 
 // ListModels : List models
-// Lists available models for Relations and Entities features, including Watson Knowledge Studio custom models that you
-// have created and linked to your Natural Language Understanding service.
+// Lists Watson Knowledge Studio [custom models](/docs/services/natural-language-understanding/customizing.html) that
```

Comment:
Erika can you refactor these relative urls to use 
https://cloud.ibm.com/docs/services/...